interface-definitions/high-availability.xml.in

```diff
@@ -117,6 +117,15 @@
                     </properties>
                     <defaultValue>60</defaultValue>
                   </leafNode>
+                  <leafNode name="ping">
+                    <properties>
+                      <help>ICMP ping health check</help>
```

Comment:
For completeness can you please also add:

```
        <valueHelp>
          <format>ipv4</format>
          <description>ICMP ping target IPv4 address</description>
        </valueHelp>
        <valueHelp>
          <format>ipv6</format>
          <description>ICMP ping target IPv6 address</description>
        </valueHelp>
```
?

Reply:
Good point! Done.